test/utils.test.js

```diff
@@ -0,0 +1,24 @@
+const { expect } = require('chai');
+const toml = require('@iarna/toml');
+const fs = require('fs');
+const { updateMoveToml, copyMovePackage } = require('../dist/utils');
+
+describe('UpdateMoveToml', () => {
+    const moveTestDir = `${__dirname}/../move-test`;
+
+    it('should insert published-at and address fields correctly', () => {
```

Comment:
wrap this in another describe block called `updateMoveToml` for better nesting

Reply:
same to above